tests/src/common/oracle.rs

```diff
@@ -57,9 +57,11 @@ impl Instantiator {
             },
             swap_tree: oracle::swap_tree!(
                 { base: StableC1::TICKER },
-                (1, PaymentC5::TICKER),
-                (3, PaymentC6::TICKER),
-                (13, PaymentC3::TICKER)
+                (1, LeaseC2::TICKER),
+                (3, LeaseC3::TICKER),
+                (7, LeaseC4::TICKER),
+                // (11, NativeC::TICKER), # FIX ME #216
+                (13, LeaseC1::TICKER),
```

Comment:
in the oracle, we do deal with payment currencies. Is there any reason I cannot see for this change?

Reply:
I have renamed them in order to be able to distinguish Lease currencies.